README.md

```diff
@@ -33,7 +33,8 @@ Clone this repository in `~/dev`.
 Open the terminal and run:
 
 ```
-$ bin/provision_osx
+$ bin/clone_homebrew
+$ bin/provision_osx link_bash_profile ## You may keep the current HEAD of the dotfiles running rather `DOTFILES_VERSION="HEAD" bin/provision_osx link_bash_profile`.
 ```
 
 Ensure the terminal uses shell `bash`, so that it uses the installed profile.
```

Comment:
```suggestion
Ensure the terminal uses shell `bash`, so that it uses the installed profile.

(If using a custom Homebrew installation, you may need to add `BREW_PREFIX=/usr/local` in `~/.bash_extra_pre`.)
```